Public/Src/FrontEnd/Script/Evaluator/Converter.cs

```diff
@@ -665,6 +665,57 @@ public static string ExtractStringLiteral(ObjectLiteral literal, SymbolAtom prop
             return ExpectNumber(value, context: new ConversionContext(name: property, objectCtx: literal));
         }
 
+        /// <summary>
+        /// Extracts optional boolean or int property.
+        /// Returns 'false' if the property is missing in a given object literal.
+        /// </summary>
+        public static bool ExtractOptionalBooleanOrInt(ObjectLiteral literal, SymbolAtom property, out bool enabled, out int intValue)

```

Comment:
>bool [](start = 22, length = 4)

I'd rather make the return type to be `(bool?, int?)?` to express the intention of extracting optional values.

Reply:
fixed in new commit